src/app_service/common/network_constants.nim

```diff
@@ -316,9 +316,6 @@ var NODE_CONFIG* = %* {
     "Enabled": true,
     "OpenseaAPIKey": OPENSEA_API_KEY_RESOLVED
   },
-  "EnsConfig": {
-    "Enabled": true
-  },
```

Comment:
why removing this?

Reply:
Good catch. Probably due to all the rebasing, I will revert it.